src/zenml/integrations/aws/__init__.py

```
@@ -42,6 +43,7 @@ class AWSIntegration(Integration):
     NAME = AWS
     REQUIREMENTS = [
         "sagemaker>=2.237.3",
+        "boto3>=1.40.30",
         "kubernetes",
         "aws-profile-manager",
     ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)